### PR TITLE
Add Flink 1.13.3 release

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

**Review by XComp — APPROVED** [review]
LGTM

File: _posts/2021-10-16-release-1.13.3.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+categories: news
+authors:
+- chesnay:
+  name: "Chesnay Schepler"
+
+---
+
+The Apache Flink community released the third bugfix version of the Apache Flink 1.13 series.
+
+This release includes 136 fixes and minor improvements for Flink 1.13.3. The list below includes bugfixes and improvements. For a complete list of all changes see:
```
Comment (leonardBang) [inline _posts/2021-10-16-release-1.13.3.md]:
```suggestion
This release includes 136 fixes and minor improvements for Flink 1.13.2. The list below includes bugfixes and improvements. For a complete list of all changes see:
```

File: _posts/2021-10-16-release-1.13.3.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients_2.11</artifactId>
+  <version>1.13.3</version>
+</dependency>
+```
+
+You can find the binaries on the updated [Downloads page]({{ site.baseurl }}/downloads.html).
+
+Below you can find more information on changes that might affect the behavior of Flink:
```
Comment (leonardBang) [inline _posts/2021-10-16-release-1.13.3.md]: Should we list all fixes and minor improvements?

**Review by leonardBang — COMMENTED** [review]
Thanks @zentol very much, I left minor comments.

File: _posts/2021-10-16-release-1.13.3.md (hunk 2 above)
Comment (zentol) [reply _posts/2021-10-16-release-1.13.3.md]: The full JIRA release notes are linked already; I don't really see the value in duplicating this information.

Comment (leonardBang) [reply _posts/2021-10-16-release-1.13.3.md]: It makes sense to me

**Pull request merged**